src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs

```diff
@@ -41,13 +42,18 @@ static Startup()
             TemplateContext.GlobalMemberAccessStrategy.Register<LocalizationPartViewModel>();
         }
 
-        public Startup(IOptions<AdminOptions> adminOptions)
+        public Startup(IConfiguration configuration, IOptions<AdminOptions> adminOptions)
```

Comment:
`IShellConfiguration`

Reply:
I was hesitate at the beginning to choose `IConfiguration` or `IShellConfiguration`, anyhow I will change it everywhere